src/vapid-helper.js

```diff
@@ -90,6 +96,48 @@ function validatePrivateKey(privateKey) {
 }
 
 /**
+ * Calculates the total expiration by adding
+ * value of Date.now() with the seconds passed
+ *
+ * @param {Number} numSeconds Number of seconds to calculate expiration
+ * @return {Number} expiration in seconds
+ */
+function calculateExpiration(numSeconds) {
+  return Math.floor(Date.now() / 1000) + numSeconds;
```

Comment:
@marco-c We don't want the time according to the local timezone of the server, we need UTC time and that's why we should use UTC equivalent methods for everything. 

The `date.setSeconds` accepts a value in the range of 0 to 59 while the number of seconds we are specifying here are going to be much more than that. Just for the sake of example
```
12 * 60 * 60 = 43200
```

Similarly `date.getSeconds` provides a value in the range of 0 to 59. Confused a bit about it. 

Here `numSeconds` is the number of seconds till the expiration which might well be a value like this one `1501807169` time in seconds 24 hours at the moment I am typing. 

I think this confusion is happening because of the confusing name, as well as the confusing comment.